modules/crm_core_match/src/Plugin/crm_core_match/engine/MatchEngineInterface.php

```diff
@@ -23,7 +23,7 @@
    * Finds matches for given contact.
    *
    * @param \Drupal\crm_core_contact\ContactInterface $contact
-   *   A contact entity used to pass data for identifying a match.
+   *   A contact interface used to pass data for identifying a match.
```

Comment:
This is not correct. It's a contact entity. There is no such thing as a contact interface.